daml-assistant/src/DAML/Assistant/Install.hs

```diff
@@ -39,16 +33,20 @@ import System.Posix.Files -- forget windows for now
 import qualified System.Info
 import qualified Data.Text as T
 import Data.Maybe
+import qualified Data.SemVer as V
+import qualified Lens.Micro as L
 
 displayInstallTarget :: InstallTarget -> Text
 displayInstallTarget = \case
-    InstallChannel (SdkChannel c) -> "channel " <> c
-    InstallVersion (SdkVersion v) -> "version " <> v
+    InstallChannel Stable -> "channel stable"
+    InstallChannel Unstable -> "channel unstable"
+    InstallChannel ch -> "channel " <> pack (show ch)
```

Comment:
```suggestion
    InstallChannel ch@(Custom _) -> "channel " <> pack (show ch)
```
Worh being explicit imho so we get a warning if we add more constructors (you could also consider changing the formatting but that’s probably not important).

Reply:
:+1: